packages/discovery-provider/src/api/v1/tracks.py

```diff
@@ -1004,14 +1004,18 @@ def get(self, track_id):
         return success_response(users)
 
 
-top_listener_response_full = full_ns.model(
+top_listener_item = ns.model(
```

Comment:
should this be `full_ns` actually?  Not sure if it matters for a nested item or not.